text/0199-support-elastic-common-schema-in-opentelemetry.md

```diff
@@ -0,0 +1,297 @@
+# Support Elastic Common Schema in OpenTelemetry
+
+Collaborators: Alexander Wert (Elastic), Jamie Hynds (Elastic), Alolita Sharma (OTel GC, Apple), Christian Beedgen (Sumo), Jonah Kowall (Logz.io), Tigran N. (Splunk), Jamie Hynds (Elastic) .
```

Comment:
Not sure if I am missing something here, but two things stood out to me in this line: 
* `Jamie Hynds (Elastic)` appears twice.
* Only @tigrannajaryan's last name is abbreviated.

Is this on purpose?